src/scoring/hpf_scoring.py

```diff
@@ -45,20 +33,18 @@
 class HPFScoring:
     """The HPF Model scoring class."""
 
-    def __init__(self, datastore=None, USE_FEEDBACK=USE_FEEDBACK):
+    def __init__(self, datastore=None, num_recommendations=5):
```

Comment:
We don't hardcode this, which is why we need the MAX_COMPANION_REC_THRESHOLD variable in the first place, so that this can be changed using the template or through the service config.